Makefile

```diff
@@ -28,6 +28,9 @@ INSTALL?=install
 # make install will place binaries here
 # the default path attempts to mimic go install
 INSTALL_DIR?=$(shell hack/build/goinstalldir.sh)
+# record the source commit in the binary
+COMMIT?=$(shell git rev-parse HEAD 2>/dev/null)
```

Comment:
Why not the short commit i

Reply:
It's easier to read imho

Reply:
The short version is just this truncated. We can truncate at runtime when formatting if we want, but this is the correct unambiguous commit ID.

Reply:
https://github.com/kubernetes-sigs/kind/pull/995